src/aspire/source/relion.py

```diff
@@ -1,20 +1,19 @@
 import logging
 import os.path
+
 from concurrent import futures
-from copy import copy
 from multiprocessing import cpu_count
 
 import mrcfile
 import numpy as np
 import pandas as pd
 
-from aspire.estimation.noise import WhiteNoiseEstimator
```

Comment:
What's happening here?

Reply:
It was imported but not used. Probably it is due to the moving of previous code.